packages/medusa/src/services/product.ts

```diff
@@ -260,6 +288,32 @@ class ProductService extends TransactionBaseService {
       )
     }
 
+    return product*/
+
+    /**
+     * TODO: The below code is a temporary fix for the issue with the typeorm idle transaction in query strategy mode
+     */
+    const manager = this.manager_
```

Comment:
todo: Use active manager
```suggestion
    const manager = this.activeManager_
```